velox/expression/Expr.cpp

```diff
@@ -1783,7 +1791,7 @@ void addStats(
   uniqueExprs.insert(&expr);
 
   // Do not aggregate empty stats.
-  if (expr.stats().numProcessedRows) {
+  if (expr.stats().numProcessedRows && !expr.stats().defaultNullRowsSkipped) {
```

Comment:
isent the default value false? should this check be this instead?

`if (expr.stats().numProcessedRows || expr.stats().defaultNullRowsSkipped)`

Since we would want to know in case numProcessedRows is zero and defaultNullRowsSkipped=true helped skip all